src/shared/parameter_v2/config_definitions/passing_config.yaml

```diff
@@ -0,0 +1,119 @@
+ - double:
+     name: static_field_position_quality_x_offset
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+                The offset from the sides of the field to place the rectangular
+                sigmoid we use to determine what areas to pass to
+ - double:
+     name: static_field_position_quality_y_offset
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+                The offset from the sides of the field to place the rectangular
+                sigmoid we use to determine what areas to pass to
+ - double:
+     name: static_field_position_quality_friendly_goal_distance_weight
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+         The weight that being close to the goal will have on the static
+         position quality. Lower, more negative weights result in the distance
+         to the goal having less of an effect.
+ - double:
+     name: enemy_proximity_importance
+     min: 0
+     max: 10
+     value: 0.5
+     description: >-
+                This controls how heavily we weight an enemy robot being near the pass
+                receiver when calculating enemy risk to a pass
+ - double:
+     name: ideal_max_rotation_to_shoot_degrees
+     min: 0
+     max: 180
+     value: 60
+     description: >-
+                The maximum angle that we have to rotate after receiving a pass to
+                shoot that we think would likely result in a goal. Note that we may
+                try to take shots that require us to rotate more then this, it's more
+                of a soft limit.
+ - double:
+     name: min_pass_speed_m_per_s
+     min: 0
+     max: 5
+     value: 3.5
+     description: "The minimum pass speed (in m/s)"
+ - double:
+     name: max_pass_speed_m_per_s
+     min: 0
+     max: 10
+     value: 5.5
+     description: "The maximum pass speed (in m/s)"
+ - double:
+     name: min_time_offset_for_pass_seconds
+     min: 0
+     max: 5
+     value: 0.7
+     description: >-
+       Minimum time into the future at which the pass should occur. This is
+       to ensure we have enough time to setup a robot to actually perform
+       the pass. This is in seconds.
+ - double:
+     name: max_time_offset_for_pass_seconds
+     min: 0
+     max: 5
+     value: 4.0
+     description: >-
+       Maximum time into the future at which the pass should occur. This gives the
+       upper bound on the pass start time, relative to the current time. This is in
+       seconds.
+ - double:
+     name: enemy_reaction_time
+     min: 0
+     max: 3.0
+     value: 0.4
+     description: >-
+       How long we think the enemy will take to recognize we're passing and start
```

Comment:
What is the value this takes if the enemy does not recognize we're passing / does not try to intercept?

Reply:
It would be 3 seconds in this case

Reply:
Is there any significance to setting the value to 0.4 instead of 3.0 in that case?

Reply:
not sure, we haven't really done a sensitivity study on these values

Reply:
I was under the impression the `value` field was a default value which is why I asked. If we're fine with it, then I don't have issues either but it might be worth noting down.

Reply:
The value field is the default, we have a _ton_ of dynamic params rn, some of which we probably dont even need. 
So we can look at the effects of this param and see if it actually does anything/tweak those 3 values or remove it altogether if it doesn't

Reply:
I feel like it makes more sense to set the default to 3.0 b/c semantically, we begin the game not knowing if they are capable of interception or not. 
Agreed, it's not a priority right now but would be nice to go through later on.

Reply:
Id argue the other way, id say its safe to assume the worst and that the enemy can _reasonably_ intercept our passes. 

And then we ease that if we are noticing that they cant

Reply:
Ah, that makes more sense when thinking that way then.